### DISCUSSION TITLE
Jak wyszukać wszystkie produkty z danej kategorii o określonych filtrach?

### DISCUSSION THREAD

**Discussion by hahyes** [allegro/allegro-api#4335] [Suggestions - rejected]
Witam,

trochę powalczyłem z wyszukiwarką produktów i nie znalazłem odpowiedzi na pytanie przez metodę prób i błędów oraz dokumentację. Czy jest możliwość wyszukiwania **wszystkich** produktów o określonej kategorii oraz filtrach? Jeżeli tak, to jak to zrobić? Z tego, co widzę, istnieje wymóg podania EAN lub frazy (przy okazji, nie oznaczyliście ich w dokumentacji jako wymagane). Czy da się wymóg ten jakoś pominąć? 

Jeżeli nie da się, sugeruję rozszerzyć wymóg o wybór trzeciej opcji - category.id. Dlaczego? Poszukuje danych i tak dla konkretnej kategorii-liścia, będącego na końcu drzewa, na dodatek z konkretnymi filtrami. Dlaczego by nie pozwolić na taką funkcjonalność? Dodatkowo, nazwy produktów są generyczne i nie znam ich formuły dla każdej z kategorii (czy jest ona powiązana z nazwą kategorii, czy może nie być?). 

Chcę wykorzystać potencjał katalogu i wyszukiwarki, jednak wymóg frazy skutecznie mi to uniemożliwia. Nie chcę przepisywać wszystkich swoich przedmiotów na produkty zawalając katalog dublami. Myślę, że jest to zrozumiałe.

Pozdrawiam.

**Comment by PrzemyslawLukanowski** [comment]
Nie da się pominąć tego wymogu, zgodnie z dokumentacją _"At least ean or phrase parameter is required_" (dlatego parametry nie są oznaczone jako wymagane w sekcji query parameters). Dziękuję za uzasadnienie potrzeby, dodałem Twoje zgłoszenie do listy naszych sugestii.

**Comment by MartaNowaczyk** [comment]
**UPDATE**: z przyczyn wewnętrznych nie planujemy wprowadzać takiej zmiany. Sugestię oznaczam jako odrzuconą.
